app/src/main/java/com/sun/uefascore/screen/topscorers/TopScorersFragment.kt

```diff
@@ -0,0 +1,22 @@
+package com.sun.uefascore.screen.topscorers
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.sun.uefascore.R
+
+class TopScorersFragment : Fragment() {
+
+    override fun onCreateView(
+            inflater: LayoutInflater, container: ViewGroup?,
+            savedInstanceState: Bundle?
+    ): View? {
```

Comment:
indent là 4 space
đọc thêm conventions về function layout nhá 
https://kotlinlang.org/docs/coding-conventions.html#formatting
```suggestion
    override fun onCreateView(
        inflater: LayoutInflater, 
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
```